packages/@webex/plugin-meetings/package.json

```diff
@@ -47,7 +47,7 @@
   },
   "dependencies": {
     "@webex/common": "workspace:^",
-    "@webex/internal-media-core": "2.0.4",
+    "@webex/internal-media-core": "link:/Users/chdubois/Projects/webrtc-media-core",
```

Comment:
please revert it to the package version number

Reply:
yes - i'm aware, I commented above

Reply:
fixed now that webrtc-media-core has been upgraded